src/Widgets/TerminalWidget.vala

```diff
@@ -155,7 +155,11 @@ namespace Terminal {
             button_release_event.connect ((event) => {
 
                 if (event.button == Gdk.BUTTON_PRIMARY) {
-                    if (allow_hyperlink) {
+                    var required_control = Application.settings.get_boolean ("control-click-open-url");
+                    var openable = required_control
+                        ? (event.state & Gdk.ModifierType.CONTROL_MASK) > 0
```

Comment:
This is `true` even if other modifiers are held down in addition to `Ctrl`.  Not sure is that matters in this case.

Reply:
several lines below the code for scrolling is
`if ((event.state & Gdk.ModifierType.CONTROL_MASK) > 0)`

I don't mind to open links with holding Ctrl + Shift, as currently no matter what modifiers it still opens up.

Reply:
Yes, I am not too concerned, although Files code does check that only `<Ctrl>` is held down.  I guess it can be changed if an issue arises.